plugins/notifications-backend/migrations/20231215_init.js

```diff
@@ -21,14 +21,13 @@ exports.up = async function up(knex) {
     table.string('title').notNullable();
     table.text('description').nullable();
     table.string('severity', 8).notNullable();
-    table.text('link').notNullable();
+    table.text('link').nullable();
```

Comment:
You need a new migration file and alterTable

Reply:
Since the SQLite does not support ALTER COLUMN and we are at the very beginning of the project (no real users so far), I think it's safe to change it here and not provide complex migration logic.

Maybe there is a better way, but such migration logic for similar case would be in creating a new table and migrating data into it for the SQLite case.

Reply:
Should be doable with knex anyways: https://github.com/knex/knex/issues/1218#issuecomment-650614036

Reply:
You are right, thank you!
There are multiple sources saying otherwise but it works this way.